evalml/pipelines/utils.py

```diff
@@ -140,6 +148,26 @@ class GeneratedPipeline(base_class):
     return GeneratedPipeline
 
 
+def get_generated_pipeline(problem_type):
+    """Returns the class for the generated pipeline based on the problem type
+
+    Arguments:
+        problem_type (Problem_Type): The problem_type that the pipeline is for
+    """
+    if problem_type == ProblemTypes.BINARY:
+        return GeneratedPipelineBinary
+    elif problem_type == ProblemTypes.MULTICLASS:
+        return GeneratedPipelineMulticlass
+    elif problem_type == ProblemTypes.REGRESSION:
+        return GeneratedPipelineRegression
+    elif problem_type == ProblemTypes.TIME_SERIES_REGRESSION:
+        return GeneratedPipelineTimeSeriesRegression
+    elif problem_type == ProblemTypes.TIME_SERIES_BINARY:
+        return GeneratedPipelineTimeSeriesBinary
+    else:
```

Comment:
Instead of having the catch all be `GeneratedPipelineTimeSeriesMulticlass`, I'd recommend checking for it specifically (`elif problem_type == ProblemTypes.TIME_SERIES_MULTICLASS:`) and then raising an error with `else` specifying that there's an unknown `ProblemTypes`